oak_offline_attestation/proto/oak_session_stream_v2.proto

```diff
@@ -0,0 +1,83 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.session.stream.v2;
```

Comment:
While this is a v2, the v1 with interactive attestation will still be needed for the Attestation Verification Service.
Will it also be remaned to v2 ?
I think we should have the attestation type (`interactive`, `non-interactive` or `offline`) in the package name.

Reply:
I think that is not going to be oak speicific, and it's also not going to be over gRPC. And when we have a key provisioning service we will not need the stream at all, so this will be renamed anyways. But I agree the package name could be more descriptive. Should I call it `oak.session.offline.stream.v1` or something like that? I don't particularly like it, but let me know if you think it clarifies things.

Reply:
Since we will have an `interactive` attestation, the counterpart could be called `noninteractive` or something else. Since the `offline` may not be descriptive enough.

Reply:
Done.